tests/Collection/ItemPriceCollectionTest.php

```diff
@@ -1,21 +1,28 @@
 <?php
+namespace Blesta\Pricing\Tests\Unit\Collection;
```

Comment:
Namespace should be `Blesta\Pricing\Tests\Collection;` *OR* test should be moved into a `Unit` directory within `tests/` (notice case).